<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageWidget</class>
 <widget class="QWidget" name="ManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>141</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="importByFile">
      <property name="text">
       <string>批量导入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="undoDeleteButton">
      <property name="text">
       <string>撤销删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DleleteAll">
      <property name="text">
       <string>删除所有选中行</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submit">
      <property name="text">
       <string>提交修改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="close">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
